esp/src/eclwatch/nls/hu/hpcc.js

```diff
@@ -207,6 +207,7 @@ define(
     IP: "IP",
     IPAddress: "IP cím",
     IsLibrary: "Könyvtár?",
+    IsReplicated: "Készült másodpéldány",
```

Comment:
In this context (IsLibrary, IsReplicated) this message should close with question mark. "Készült másodpéldány?"